Dockerfile

```diff
@@ -27,6 +27,9 @@ COPY controllers/ controllers/
 COPY internal/ internal/
 COPY pkg/ pkg/
 
+# Required to include the build information into the binary
+COPY .git .git
```

Comment:
Is there a reason we don't just run `COPY ./ ./` instead of copying all the relevant files one by one?
Even if we don't need all of them, this is only the build container so it doesn't affect the final image and we won't need to copy each on of the files each time there is a new requirement.
WDYT?

Reply:
I think listing everything we need avoids side effects, but I don't have a very strong opinion there.

Reply:
Side effects aren't always bad, one of the side effect could have been that you get "for free" and the new `.git` dependency to the container.

But again, I don't really mind.